packages/lit-dev-content/samples/tutorials/svg-templates/05/after/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script type="module" src="repeat-pattern.js"></script>
+    <style>
+      html, body {
+         height: 100%;
+         padding: 0;
+         margin: 0;
+      }
+      body {
+        --background-color: #ffa500;
+        --font-color: #000;
+        --font-size: 26px;
+        --stroke-width: 1.1;
+        --stroke-color: #0000dd;
+      }
+    </style>
+  </head>
+  <body>
+    <repeat-pattern num-prints="12"></repeat-pattern>
```

Comment:
i think `num-prints` is not supposed to be here

Reply:
done!